Make format does not format all files

The `make format` target currently only formats the first one found. This ought to be fixed so that it formats all.

I get the following error with `make format`

```
rm -f decide/*.o DECIDE decide/*.test
rm: DECIDE: is a directory
make: *** [clean] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ DECIDE: $(objects) decide/main.o
 
 ### Formatting
 format: $(wildcard decide/*.?pp)
-	clang-format -i $< --style=file
+	clang-format -i $^ --style=file
 
 
 ### Testing

